pkg/driver/netnamespace.go

```diff
@@ -131,3 +132,46 @@ func applyNeighborConfig(containerNsPAth string, ifName string, neighConfig []ap
 	}
 	return errors.Join(errorList...)
 }
+
+func applyRulesConfig(containerNsPath string, rulesConfig []apis.RuleConfig) error {
+	containerNs, err := netns.GetFromPath(containerNsPath)
+	if err != nil {
+		return err
+	}
+	defer containerNs.Close()
+
+	nhNs, err := nlwrap.NewHandleAt(containerNs)
```

Comment:
nit: Can we make this a more understandable name for the variable? nhNs is difficult to understand.

Reply:
Sure done